### PR TITLE
color correction of placeholder

### PR DESCRIPTION
**Pull request by shreya-fyle1**

### CODE REVIEW

File: src/app/shared/components/fy-number/fy-number.component.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
     margin: 4px 0;
     padding: 0;
     &::placeholder {
-      color: $blue-black !important;
+      color: #a9acbc !important;
```
Comment (yksolanki9) [inline src/app/shared/components/fy-number/fy-number.component.scss:15]: Check the colors name in `colors.scss` and use name instead of the hex value

**Review by yksolanki9 — CHANGES_REQUESTED** [review]

**Review by yksolanki9 — APPROVED** [review]
Test all number fields in expense/mileage/per-diem/advances before merging

**Review by Dimple16 — APPROVED** [review]

**Comment by shreya-fyle1** [comment]
> Test all number fields in expense/mileage/per-diem/advances before merging

yes done.

**Comment by shreya-fyle1** [comment]
image

**Comment by shreya-fyle1** [comment]
image

**Comment by shreya-fyle1** [comment]
> Test all number fields in expense/mileage/per-diem/advances before merging

Yes Done

**Pull request merged**